src/app/app.component.html

```diff
@@ -1,5 +1,4 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
```

Comment:
Why did you remove div? Maybe you want to put it back we will need it.

Reply:
I don't even know what that is?